packages/tree/src/Dendrogram.ts

```diff
@@ -154,14 +155,34 @@ export class Dendrogram extends SVGZoomWidget {
         lines.exit().remove();
 
         //  Nodes  ---
-        const textOffsetX = this.circleRadius() + 2;
+        const textOffsetX = context.labelInNode() ? 0 : this.circleRadius() + 2;
         function nodeTransform(d) {
             if (context.radial()) {
                 return "rotate(" + (d.x - 90) + ")translate(" + d.y + ")";
             }
             return context.orientation() === "horizontal" ? "translate(" + d.y + "," + d.x + ")" : "translate(" + d.x + "," + d.y + ")";
         }
+        const sizes = dataNodes
+            .map(n => {
+                return _findSize(n);
+
+                function _findSize(node) {
+                    if (typeof node.data.size !== "undefined") {
+                        return node.data.size;
+                    } else if (typeof node.children !== "undefined" && node.children.length > 0) {
+                        let ret = 0;
+                        for (const child of node.children) {
+                            ret += _findSize(child);
+                        }
+                        return ret;
+                    }
+                    return 0;
+                }
+            })
+            .filter(n => n !== null)
```

Comment:
When can n be null?

Reply:
Good catch, that was left over from an earlier version of that size mapping.